docs/user/README.md

```diff
@@ -1,44 +1,67 @@
-# Glean SDK
+# Introduction
 
-![Glean logo](glean.jpeg)
+Glean is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+## Contact
 
-The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+To contact the Glean team you can:
 
-To contact us you can:
 - Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
-- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
 - Send an email to *glean-team@mozilla.com*.
 - The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
 
-The source code is available [on GitHub](https://github.com/mozilla/glean/).
+![Glean logo](glean.jpeg)
```

Comment:
What was the reason for moving the logo down here?
IMO it was good as the big splash to start with.

Plus now the contact info is front and center and I'm not sure that's the best way

Reply:
Yeah, I had put the contact before the `## Sections` title, but I was torn... I can move it back there, or somewhere else. The truth is I didn't know where to put it :D

Reply:
Side-note: do we want the logo with a white or transparent background to be shown here?

Reply:
> Side-note: do we want the logo with a white or transparent background to be shown here?

I thought about this. The black adds a bit of drama, which I appreciate. IMO, we should keep it black.

Reply:
(I moved the `Contact` section to right before the `License` section in this commit https://github.com/mozilla/glean/pull/1528/commits/e45e44225e54cf570489499c1c9928d2dbab72eb, still not 100% secure about its positioning)